src/section/section_map.rs

```diff
@@ -346,13 +308,15 @@ impl SectionMap {
         let _ = self.keys.insert(prefix, new_key);
     }
 
-    pub fn get_knowledge(&self, prefix: &Prefix<XorName>) -> Option<u64> {
-        self.knowledge.get(prefix).copied()
+    /// Returns the the index of the public key in our_history that will be trusted by the given
+    /// section.
+    pub fn knowledge_by_section(&self, prefix: &Prefix<XorName>) -> u64 {
+        self.knowledge.get(prefix).copied().unwrap_or(0)
```

Comment:
just to ensure that `index 0` won't point to a valid entry, right?

Reply:
Yes, index `0` points to the genesis key which every section knows.